README.md

```diff
@@ -1,5 +1,5 @@
 # Eclipse Che documentation project
-[![Contribute](https://www.eclipse.org/che/contribute.svg)](https://che.openshift.io/#https://github.com/eclipse/che-docs)
+[![Contribute](https://www.eclipse.org/che/contribute.svg)](https://workspaces.openshift.com/#https://github.com/eclipse/che-docs)
```

Comment:
@themr0c this is an incorrect reference to factory - should be based on `f?url=` see #1884